Sources/SwiftSMTP/Sender.swift

```diff
@@ -149,7 +149,7 @@ extension String {
             throw SMTPError(.createEmailRegexFailed)
         }
 
-        let range = NSRange(location: 0, length: self.characters.count)
+        let range = NSRange(location: 0, length: self.count)
```

Comment:
You could remove `self` in `self.count` here if you wanted.

Reply:
Does IBM have any Swift style guidelines I could follow in the future? I understand that Swift typically advises against using "self", but didn't change it in this instance as it was used in "self.characters.count".

Reply:
In general we've followed https://github.com/raywenderlich/swift-style-guide although it's a bit out of date now.  Removing `self.` in this case is a good idea :)